esphome/dashboard/web_server.py

```diff
@@ -52,6 +52,8 @@
     from requests import Response
 
 
+LOG_LEVEL = os.getenv("LOG_LEVEL", "INFO").upper()
```

Comment:
I think we should prefix this
```suggestion
LOG_LEVEL = os.getenv("ESPHOME_LOG_LEVEL", "INFO").upper()
```

Reply:
Makes perfect sense to me